fontra2fontir/src/fontra.rs

```diff
@@ -141,19 +141,83 @@ impl FontraFontData {
 }
 
 #[derive(Debug, Clone, Deserialize)]
-pub(crate) struct FontraAxis {
+#[serde(untagged)]
+pub(crate) enum FontraAxis {
+    Continuous(FontraContinuousAxis),
+    Discrete(FontraDiscreteAxis),
+}
+
+#[allow(dead_code)] // TEMPORARY
+impl FontraAxis {
+    pub(crate) fn name(&self) -> &AxisName {
+        match self {
+            FontraAxis::Continuous(a) => &a.name,
+            FontraAxis::Discrete(a) => &a.name,
+        }
+    }
+
+    pub(crate) fn tag(&self) -> Tag {
+        match self {
+            FontraAxis::Continuous(a) => a.tag,
+            FontraAxis::Discrete(a) => a.tag,
+        }
+    }
+
+    pub(crate) fn default_value(&self) -> f64 {
+        match self {
+            FontraAxis::Continuous(a) => a.default_value,
+            FontraAxis::Discrete(a) => a.default_value,
+        }
+    }
+
+    pub(crate) fn hidden(&self) -> bool {
+        match self {
+            FontraAxis::Continuous(a) => a.hidden,
+            FontraAxis::Discrete(a) => a.hidden,
+        }
+    }
+
+    pub(crate) fn mapping(&self) -> &Vec<[f64; 2]> {
```

Comment:
This I think warrants a comment on how to interpret each pair, e.g. interpret as [user, design] forming a piecewise linear map